src/main/java/dsn/package-info.java

```diff
@@ -2,12 +2,14 @@
 @WithMappers(BasicValueMappers.class)
 @WithConfiguration(Configuration.class)
 @WithActivityType(DSN_Track.class)
-@WithActivityType(DSN_View_Period.class)
+@WithActivityType(DSN_View_Period_Event.class)
```

Comment:
Could you remind me what the rationale was for having a separate "Event" activity and "Duration" activity? No objections at this time, just would like to understand